requirements-dev.txt

```diff
@@ -1,6 +1,9 @@
-ipython
-requests_cache
-sphinx
-sphinx_rtd_theme
-black
-isort
+black; python_version > '3.5'
+flake8-bugbear; python_version > '3.5'
```

Comment:
We install ``flake8-bugbear`` in CI, but are we using it anywere?

Reply:
My understanding is that ```flake8-bugbear``` is a plugin into ```flake8```, so it is a development dependency

Reply:
bugbear is optional and can't be a dependency since it doesn't support Python 2.7.  It can go in after this release.

Reply:
The flake8 'B' tests are bugbear.

Reply:
the specific syntax above should only install it for ```3.5+```

Reply:
Ignore my previous.  This method is fine.